test/example.js

```diff
@@ -254,12 +258,33 @@ PrototypeClass.method2 = function(num) {};
 //   `Uint32Array`,
 //   `Float32Array`,
 //   `Float64Array`,
+//
 //   `Error`,
 //   `EvalError`,
-//   `TypeError`,
 //   `RangeError`,
-//   `SyntaxError`,
 //   `ReferenceError`,
+//   `SyntaxError`,
+//   `TypeError`,
+//   `URIError`,
+//
+//   `Intl.Collator`,
+//   `Intl.DateTimeFormat`,
+//   `Intl.ListFormat`,
+//   `Intl.NumberFormat`,
+//   `Intl.PluralRules`,
+//   `Intl.RelativeTimeFormat`,
+//
+//   `WebAssembly.Global`,
+//   `WebAssembly.Module`,
+//   `WebAssembly.Instance`,
+//   `WebAssembly.Memory`,
+//   `WebAssembly.Table`,
+//   `WebAssembly.CompileError`,
+//   `WebAssembly.LinkError`,
+//   `WebAssembly.RuntimeError`,
+//
+//   `Primitive`,
+//   `Iterable`,
```

Comment:
What about the `example.md`?

Reply:
This comment never belonged to any function thus documentation is not generated for it.

Reply:
Perhaps it's better to document it (add to some function) to check that correct links are generated?

Reply:
In this case, we should move it to `header.md` and remove it from here.

Reply:
As for now `metadoc` will trim excessive whitespaces including newlines to correctly generate line length of which won't exceed 80 characters. This means that lists like this will be trimmed and the generated doc will look like this:

```md
Multi-line expanded function description. Note that all lines should have length
of 80 or less characters. To specify custom arguments signature `Signature:`
comment can be used. It also supports multiline. Backticks `are supported` and
they `` can be used `inside` code blocks ``. Note
that`text that is not`separated from backtick will stick to it if line exceeds
max length. List of supported standard types: [`<boolean>`][boolean],
[`<null>`][null], [`<undefined>`][undefined], [`<number>`][number],
[`<string>`][string], [`<symbol>`][symbol], [`<Object>`][object],
[`<Date>`][date], [`<BigInt>`][bigint], [`<Function>`][function],
[`<RegExp>`][regexp], [`<Promise>`][promise], [`<DataView>`][dataview],
[`<Proxy>`][proxy], [`<Map>`][map], [`<WeakMap>`][weakmap], [`<Set>`][set],
[`<WeakSet>`][weakset], [`<Array>`][array], [`<ArrayBuffer>`][arraybuffer],
[`<Int8Array>`][int8array], [`<Uint8Array>`][uint8array],
[`<Uint8ClampedArray>`][uint8clampedarray], [`<Int16Array>`][int16array],
[`<Uint16Array>`][uint16array], [`<Int32Array>`][int32array],
[`<Uint32Array>`][uint32array], [`<Float32Array>`][float32array],
[`<Float64Array>`][float64array], [`<Error>`][error],
[`<EvalError>`][evalerror], [`<RangeError>`][rangeerror],
[`<ReferenceError>`][referenceerror], [`<SyntaxError>`][syntaxerror],
[`<TypeError>`][typeerror], [`<URIError>`][urierror],
[`<Intl.Collator>`][intl.collator],
[`<Intl.DateTimeFormat>`][intl.datetimeformat],
[`<Intl.ListFormat>`][intl.listformat],
[`<Intl.NumberFormat>`][intl.numberformat],
[`<Intl.PluralRules>`][intl.pluralrules],
[`<Intl.RelativeTimeFormat>`][intl.relativetimeformat],
[`<WebAssembly.Global>`][webassembly.global],
[`<WebAssembly.Module>`][webassembly.module],
[`<WebAssembly.Instance>`][webassembly.instance],
[`<WebAssembly.Memory>`][webassembly.memory],
[`<WebAssembly.Table>`][webassembly.table],
[`<WebAssembly.CompileError>`][webassembly.compileerror],
[`<WebAssembly.LinkError>`][webassembly.linkerror],
[`<WebAssembly.RuntimeError>`][webassembly.runtimeerror],
[`<Primitive>`][primitive], [`<Iterable>`][iterable], [`<this>`][this] List of
supported non-standard types: `<any>`

...

[meta doc]: https://github.com/metarhia/metadoc
[introspect()]: https://github.com/metarhia/metadoc#introspectnamespace-text
[object]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
[date]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
[bigint]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
[function]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function
[regexp]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
[promise]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
[dataview]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView
[proxy]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy
[map]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
[weakmap]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap
[set]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
[weakset]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet
[array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
[arraybuffer]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer
[int8array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int8Array
[uint8array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array
[uint8clampedarray]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8ClampedArray
[int16array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int16Array
[uint16array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint16Array
[int32array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int32Array
[uint32array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint32Array
[float32array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Float32Array
[float64array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Float64Array
[error]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error
[evalerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/EvalError
[rangeerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError
[referenceerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError
[syntaxerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError
[typeerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError
[urierror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError
[intl.collator]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Collator
[intl.datetimeformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat
[intl.listformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat
[intl.numberformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
[intl.pluralrules]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules
[intl.relativetimeformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat
[webassembly.global]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Global
[webassembly.module]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Module
[webassembly.instance]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Instance
[webassembly.memory]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory
[webassembly.table]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Table
[webassembly.compileerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/CompileError
[webassembly.linkerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/LinkError
[webassembly.runtimeerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/RuntimeError
[boolean]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type
[null]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Null_type
[undefined]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Undefined_type
[number]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type
[string]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type
[symbol]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Symbol_type
[primitive]: https://developer.mozilla.org/en-US/docs/Glossary/Primitive
[iterable]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols
[this]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this
```
which will be rendered like this:

Multi-line expanded function description. Note that all lines should have length
of 80 or less characters. To specify custom arguments signature `Signature:`
comment can be used. It also supports multiline. Backticks `are supported` and
they `` can be used `inside` code blocks ``. Note
that`text that is not`separated from backtick will stick to it if line exceeds
max length. List of supported standard types: [`<boolean>`][boolean],
[`<null>`][null], [`<undefined>`][undefined], [`<number>`][number],
[`<string>`][string], [`<symbol>`][symbol], [`<Object>`][object],
[`<Date>`][date], [`<BigInt>`][bigint], [`<Function>`][function],
[`<RegExp>`][regexp], [`<Promise>`][promise], [`<DataView>`][dataview],
[`<Proxy>`][proxy], [`<Map>`][map], [`<WeakMap>`][weakmap], [`<Set>`][set],
[`<WeakSet>`][weakset], [`<Array>`][array], [`<ArrayBuffer>`][arraybuffer],
[`<Int8Array>`][int8array], [`<Uint8Array>`][uint8array],
[`<Uint8ClampedArray>`][uint8clampedarray], [`<Int16Array>`][int16array],
[`<Uint16Array>`][uint16array], [`<Int32Array>`][int32array],
[`<Uint32Array>`][uint32array], [`<Float32Array>`][float32array],
[`<Float64Array>`][float64array], [`<Error>`][error],
[`<EvalError>`][evalerror], [`<RangeError>`][rangeerror],
[`<ReferenceError>`][referenceerror], [`<SyntaxError>`][syntaxerror],
[`<TypeError>`][typeerror], [`<URIError>`][urierror],
[`<Intl.Collator>`][intl.collator],
[`<Intl.DateTimeFormat>`][intl.datetimeformat],
[`<Intl.ListFormat>`][intl.listformat],
[`<Intl.NumberFormat>`][intl.numberformat],
[`<Intl.PluralRules>`][intl.pluralrules],
[`<Intl.RelativeTimeFormat>`][intl.relativetimeformat],
[`<WebAssembly.Global>`][webassembly.global],
[`<WebAssembly.Module>`][webassembly.module],
[`<WebAssembly.Instance>`][webassembly.instance],
[`<WebAssembly.Memory>`][webassembly.memory],
[`<WebAssembly.Table>`][webassembly.table],
[`<WebAssembly.CompileError>`][webassembly.compileerror],
[`<WebAssembly.LinkError>`][webassembly.linkerror],
[`<WebAssembly.RuntimeError>`][webassembly.runtimeerror],
[`<Primitive>`][primitive], [`<Iterable>`][iterable], [`<this>`][this] List of
supported non-standard types: `<any>`

[meta doc]: https://github.com/metarhia/metadoc
[introspect()]: https://github.com/metarhia/metadoc#introspectnamespace-text
[object]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
[date]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
[bigint]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
[function]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function
[regexp]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp
[promise]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
[dataview]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView
[proxy]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy
[map]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
[weakmap]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap
[set]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set
[weakset]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet
[array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
[arraybuffer]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer
[int8array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int8Array
[uint8array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array
[uint8clampedarray]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8ClampedArray
[int16array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int16Array
[uint16array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint16Array
[int32array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Int32Array
[uint32array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint32Array
[float32array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Float32Array
[float64array]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Float64Array
[error]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error
[evalerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/EvalError
[rangeerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError
[referenceerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ReferenceError
[syntaxerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SyntaxError
[typeerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError
[urierror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/URIError
[intl.collator]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Collator
[intl.datetimeformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat
[intl.listformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat
[intl.numberformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat
[intl.pluralrules]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules
[intl.relativetimeformat]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RelativeTimeFormat
[webassembly.global]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Global
[webassembly.module]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Module
[webassembly.instance]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Instance
[webassembly.memory]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory
[webassembly.table]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Table
[webassembly.compileerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/CompileError
[webassembly.linkerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/LinkError
[webassembly.runtimeerror]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/RuntimeError
[boolean]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type
[null]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Null_type
[undefined]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Undefined_type
[number]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type
[string]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type
[symbol]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Symbol_type
[primitive]: https://developer.mozilla.org/en-US/docs/Glossary/Primitive
[iterable]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Iteration_protocols
[this]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this

Reply:
We can move them somewhere in `header.md` as just plain text and have an example for each different type of link. Also, I'm not a fan of the idea of having them all even in the test file. Not mentioning the fact that we won't be adding new types very often.

Reply:
I think we can extract this in a separate PR for that, and land this as is.